src/cascadia/TerminalApp/App.cpp

```diff
@@ -428,6 +428,8 @@ namespace winrt::TerminalApp::implementation
         bindings.ScrollDownPage([this]() { _ScrollPage(1); });
         bindings.SwitchToTab([this](const auto index) { _SelectTab({ index }); });
         bindings.OpenSettings([this]() { _OpenSettings(); });
+        bindings.CopyText([this]() { _CopyText(true);  });

```

Comment:
2 things:
 1) nit: extra space on the right side of the first semicolon
 2) If I recall correctly, the parameter for `_CopyText()` decides whether or not to copy new lines (usually determined by the SHIFT button). So we'll have to detect that too and use that for this parameter (we'll probably need to make that button rebindable too).